core/src/main/java/nl/nn/adapterframework/configuration/classloaders/ClassLoaderBase.java

```diff
@@ -74,16 +74,17 @@ public void configure(IbisContext ibisContext, String configurationName) throws
 		} else {
 			if(basePath == null && !getConfigurationName().equalsIgnoreCase(instanceName)) {
 				int i = configurationFile.lastIndexOf('/');
-				if (i != -1) {
+				if (i != -1) { //Configuration file contains a path, derive the BasePath from the path
 					setBasePath(configurationFile.substring(0, i + 1));
```

Comment:
Waarom heeft ClassLoaderBase nog steeds een basePath? Hij heeft het zelf toch niet nodig?

Reply:
Legacy, alle classloaders hebben een root die gelijk is aan de configuratie naam. En mocht je via de configurationFile propertie een path+file combinatie opgeven, dan is het 'path' gedeelte de root van de classloader (aka basepath).